Core/Frameworks/BaikalAdmin/Controller/Settings/System.php

```diff
@@ -129,6 +129,13 @@ function validationHook(\Formal\Form $oForm, \Formal\Form\Morphology $oMorpho) {
                 // We'll have to clean this up later.
                 $sFile = eval('return ' . $sFile . ';');
 
+                if (!file_exists($sFile)) {
+                    $sMessage = "DB file does not exist.";
+                    $sMessage .= "</br>You can copy a sqlite Database file from there: <strong>Core/Resources/Db/SQLite/db.sqlite</strong>";
```

Comment:
This is incorrect actually.. that file no longer exists, because it was actually a bad idea to ship a sqlite database, as it doesn't work for every version of sqlite. Furthermore, the sqlite database should get automatically created.